src/HealthGPS.Core/datastore.h

```diff
@@ -49,35 +51,35 @@ class Datastore {
     /// @brief Gets a disease full definition by identifier for a country
     /// @param info The target disease information
     /// @param country The target country definition
-    /// @return The disease definition, check empty() = true for missing data.
+    /// @return The disease definition
     virtual DiseaseEntity get_disease(const DiseaseInfo &info, const Country &country) const = 0;
 
     /// @brief Gets the relative risk effects for disease to disease interactions
     /// @param source The source disease information
     /// @param target The target disease information
-    /// @return The disease-disease effects, check empty() = true for missing data.
+    /// @return The disease-disease effects
     virtual RelativeRiskEntity get_relative_risk_to_disease(const DiseaseInfo &source,
                                                             const DiseaseInfo &target) const = 0;
```

Comment:
It should also be possible to apply the `optional` approach from `get_relative_risk_to_risk_factor` here.